### PR TITLE
Removed django.contrib.gis.geometry.backend.

### PR DESCRIPTION
**Pull request by timgraham**
The layer of indirection is unchanged and undocumented
since its introduction in ff60c5f9de3e8690d1e86f3e9e3f7248a15397c8.

### CODE REVIEW

**Comment by timgraham** [comment]
@sir-sigurd, @claudep, @jbronn (maybe you authored this code?) -- do you see any use for this?

**Comment by claudep** [comment]
It might have been a bit overengineered at the time. I never encountered a usage of `GEOMETRY_BACKEND` until now.

**Comment by jbronn** [comment]
The original intention was so that other geometry wrappers could be used -- in particular Shapely or GDAL's `OGRGeometry` (memory is hazy, but OGR may be faster at parsing WKT).  This vision was never realized, and I'm fine with removing the dead over-engineered code I introduced.

**Comment by ngnpope** [comment]
Any reason to leave the module `django.contrib.gis.geometry.backend.geos` behind? Seems unnecessary to have the aliases as you've gone through and replaced (all of?) them with direct imports from `django.contrib.gis.geos`.

And I suppose if `django.contrib.gis.geometry.regex` is all that is left and is non-public, it could possibly be merged into `django.contrib.gis.geos`?

**Comment by timgraham** [comment]
Sure Nick, leaving `backend.geos` wasn't intentional. I added another commit to move `gis.geometry.regex` to `gi.geometry` (other name suggestions welcome) -- it's shared between geos and gdal.

**Comment by ngnpope** [comment]
Looks good, Tim. Thanks.

File: repeated-django-test.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+for i in {1..500};
```
Comment (sir-sigurd) [inline repeated-django-test.sh]: Is that added accidentally?

**Pull request merged**